pages/en/lb4/Decorators.md

```diff
@@ -354,7 +354,7 @@ For more information, see the [Dependency Injection](Dependency-Injection.htm) s
 
   Model decorator is a Class decorator.
   In LoopBack 4, we inherit the model definition format from LoopBack 3,
-  you can find it in [Model definition JSON file](/lb3/Model-definition-JSON-file.html).
+  you can find it in [Model definition JSON file](https://loopback.io/doc/en/lb3/Model-definition-JSON-file.html).
```

Comment:
I think we should be using relative paths. That way the links will keep working after we clone content, e.g. as had been done in the past when we created `lb3` as a copy of `lb2`.

```diff
-  you can find it in [Model definition JSON file](/lb3/Model-definition-JSON-file.html).
+  you can find it in [Model definition JSON file](Model-definition-JSON-file.html).
```

Reply:
I thought that we should be using relative paths as well. But since appending `/lb3` breaks the link, should we be creating a copy of the file for `lb4`?

Reply:
Can you try `../lb3/...`

Reply:
Wow, that worked. I'll be creating a PR to apply this fix

Reply:
Oh, I did not realize this is a `lb4` file referring to `lb3` content. I like @virkt25's proposal 👍